derive_builder/src/options/darling_opts.rs

```diff
@@ -319,6 +319,16 @@ impl Options {
             .fields
     }
 
+    /// A builder requires `Clone` to be derived if any of its fields use the mutable or immutable pattern.
+    ///
+    /// This can technically be expressed two ways:
```

Comment:
Feels weird to explain how to express that `Clone` is required, and it should be the non-owned patterns that cause `Clone` to be required. I think the first sentence by itself is enough

Reply:
You're right; I'll remove the second part of this comment.